workflows/Delete_O365_Emails_with_Slack/workflow.spec.yaml

```
@@ -0,0 +1,27 @@
+extension: workflow
+products: ["insightconnect"]
+name: Delete_O365_Emails_With_Slack
+title: "Delete O365 Emails With Slack"
+description: "This workflow will delete emails matching given search criteria by sending a notification to InisghtConnect with Slack. The workflow will look for any emails that match the body, subject, for from contents given. It can either automatically delete these emails or prompt the user for further action."
```

Short-description here
```suggestion
description: "This workflow will delete emails matching given search criteria by sending a notification to InisghtConnect with Slack."
```